config/yurtctl-servant/Dockerfile

```diff
@@ -1,4 +1 @@
-FROM alpine:3.8
```

Comment:
When the pod Yurtctl-servant-convert runs the command  `nsenter -t 1 -m -u -n -i /bin/yurtctl convert edgenode --yurthub-image {{.yurthub_image}} --join-token {{.joinToken}} --pod-manifest-path {{.pod_manifest_path}}` , command  `nsenter` in alpine will parse `--yurthub-image` into the parameters of nsenter. It will not have this problem with ubuntu.

Reply:
Thanks for the explanation. Would you try `nsenter -t 1 -m -u -n -i  -- /bin/yurtctl convert edgenode --yurthub-image {{.yurthub_image}} --join-token {{.joinToken}} --pod-manifest-path {{.pod_manifest_path}}` in alpine? Let's use '--' to tell nsenter to stop option parsing. As the size of the alpine is smaller than the ubuntu, using it can help us reduce the network throughput when setting up a large cluster.

Reply:
Thanks！I test it and it works！